src/redux/modules/auth/middleware.js

```diff
@@ -0,0 +1,22 @@
+import AuthService from '../../../utils/AuthService';
+
+import { loginSuccess } from './actions';
+
+const authMiddleware = store => next => (action) => {
+  AuthService.handleAuthentication()
+    .then((logged) => {
+      window.location.href = '/';
+      store.dispatch(loginSuccess(logged));
+      next(action);
+    }).catch(() => {
+      // this is actually not true error
+      // console.log('err', err);
+      const isAuthenticated = AuthService.isAuthenticated();
+      if (isAuthenticated) {
+        console.log('getProfile: ', AuthService.getProfile());
```

Comment:
`console.log` to be removed?

Reply:
we will remove this once we stabilize Auth0, i have updated to note this although